drivers/axi_core/jesd204/axi_jesd204_rx.c

```diff
@@ -311,6 +350,48 @@ int32_t axi_jesd204_rx_laneinfo_read(struct axi_jesd204_rx *jesd, uint32_t lane)
 	return SUCCESS;
 }
 
+/**
+ * @brief axi_jesd204_rx_laneinfo_64b66b_read
+ */
+static int32_t axi_jesd204_rx_laneinfo_64b66b_read(struct axi_jesd204_rx *jesd,
+		uint32_t lane,
+		uint32_t lane_status)
+{
+	uint8_t extend_multiblock;
+
+	extend_multiblock = JESD204_EMB_STATE_GET(lane_status);
+
+	printf("\tState of Extended multiblock alignment: %s\n",
```

Comment:
Is it OK for drivers to try to print to the output stream?

Reply:
On the platforms where JESD is used, printf is redirected to UART.